### DISCUSSION TITLE
Will the .html(history) file disappear if the account is deleted?

### DISCUSSION THREAD

**Discussion by zerakiel** [Tyrrrz/DiscordChatExporter#898] [Q&A]
might be a stupid question but since i didnt find a answer, im aware that its a .html file but im not entirely sure that the messages will be deleted once the discord user is gone?

**Comment by 96-LB** [comment]
Once the file is created, it won't be modified in any way. If you didn't turn on the download media option, the links to user avatars/attachments may stop working when you delete the accounts/channel, but message history will be preserved. If you download media then the images will be safe too.
